workspaces/arborist/lib/shrinkwrap.js

```diff
@@ -134,81 +112,72 @@ const nodeMetaKeys = [
 
 const metaFieldFromPkg = (pkg, key) => {
   const val = pkg[key]
-  // get the license type, not an object
-  return (key === 'license' && val && typeof val === 'object' && val.type)
-    ? val.type
+  if (val) {
+    // get only the license type, not the full object
+    if (key === 'license' && typeof val === 'object' && val.type) {
```

Comment:
```suggestion
    if (key === 'license' && val?.type) {
```

Reply:
I kept this in for future me reading this to remember that the "object" format of the license is what we're after here. For better or worse.